utils/fakeuname/Makefile

```diff
@@ -0,0 +1,56 @@
+# Copyright (C) 2022 Sergey V. Lobanov <sergey@lobanov.in>
+#
+# SPDX-License-Identifier: GPL-2.0-or-later
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=fakeuname
+PKG_RELEASE:=$(COMMITCOUNT)
+PKG_LICENSE:=GPL-2.0-or-later
+
+PKG_MAINTAINER:=Sergey V. Lobanov <sergey@lobanov.in>
+
+HOST_BUILD_DEPENDS=linux
+PKG_HOST_ONLY:=1
+
+define Package/fakeuname
+  SECTION:=utils
+  CATEGORY:=Utilities
+  TITLE:=Fake uname host tool for cross-compile purposes
+  BUILDONLY:=1
+endef
+
+define Package/fakeuname/description
+ Fakeuname is a host tool for cross-compile cross-platform builds
+ if configure or/and build scripts check uname output for target 
+ build and use invalid build flags. This tool should not be used 
+ in normal case if configure/build scripts allow to redefine 
+ required values instead of using uname output
+endef
+
+include $(INCLUDE_DIR)/kernel.mk
+include $(INCLUDE_DIR)/package.mk
+include $(INCLUDE_DIR)/host-build.mk
+
+define Host/Configure
+endef
+
+define Host/Compile
+	$(CP) src/header.py.inc $(HOST_BUILD_DIR)/$(PKG_NAME)
+	echo machine = \"$(ARCH)\" >> $(HOST_BUILD_DIR)/$(PKG_NAME)
```

Comment:
these look like they can be batched.

Reply:
Sorry, but I didn't get your idea. Could you please explain?

Reply:
```
{
echo a
echo b
} >> $(HOST_BUILD_DIR)/$(PKG_NAME)
```

Reply:
reworked as requested